### ISSUE TITLE
Remove Pause and Resume feature

### ISSUE THREAD

**Issue by PtrckM** [hummingbot/hummingbot#4428]
**Describe the bug**
When using the feature [Pause and Resume strategy ](https://docs.hummingbot.io/operation/pause-resume/), the `exit --suspend` and `start --restore` not working as intended. On source build bot able to detect the order but immediately cancel it, but on binary and docker bot not able to detect it. Upon stop command bot will cancel all orders created by the bot.

**Steps To Reproduce**

* create strategy using pure_mm and exchange (binance, gate_io, kucoin, crypto_com, ascendex)
* select your pair
* start strategy
* after the bot created your order do `exit --suspend`
* check the exchange if orders are there
* restart the bot then import your strategy again then `start --restore`

**Screenshots**

* `exit --suspend`
![image](https://user-images.githubusercontent.com/54516858/136502338-3189f708-7b53-4102-81e1-2b725986ccbc.png)
![image](https://user-images.githubusercontent.com/54516858/136502359-1208c1cc-32bf-4b35-8b3e-95176e6a16b2.png)
* `import` then `start --restore` 
![](https://user-images.githubusercontent.com/54516858/136502431-60507b63-358d-4cac-904a-d3dab1720ec2.png)

![image](https://user-images.githubusercontent.com/54516858/136502411-4c86c50e-f5ce-43c7-b118-cda2c83d864e.png)

* `stop` the bot able to find all the orders created by the bot.
![image](https://user-images.githubusercontent.com/54516858/136502501-5ce29ca0-0dc4-41ee-8df2-85ad7cbde4a9.png)

**Release version**
0.44.0 

**Attachments**
[logs_kucoin.log](https://github.com/CoinAlpha/hummingbot/files/7308735/logs_kucoin.log)

[kucoin_strategy.txt](https://github.com/CoinAlpha/hummingbot/files/7308737/kucoin_strategy.txt)

┆Issue is synchronized with this [Clickup task](https://app.clickup.com/t/1pd0fpx) by [Unito](https://www.unito.io)

**Comment by PtrckM** [comment]
adding Gate_io to the list of affected exchanges (version 0.45.0) Source build

**Comment by PtrckM** [comment]
adding mexc to the list of affected exchanges (version 1.4.0)

**Comment by rapcmia** [comment]
Severity: P3
Bounty: 10,000 (https://hummingbot.org/maintenance/bugs/)
Rationale: Bug related to component for Pure market making

**Comment by MementoRC** [comment]
Looking at the code, this is the logic implemented, the `restore` option for `start` prevents the cancellation of existing orders on the exchange. If started without this option, the bot would cancel those orders

**Comment by nikspz** [comment]
@MementoRC 

Bot should left orders uncancelled by exit --suspend

Then this option should allow bot to restore it (see that uncancelled orders) 
https://docs.hummingbot.org/operation/pause-resume/#pause-and-resume-strategy

Currently bot didn't show it as hanging orders in the status (it creates new pair of orders and only this new showed on `status`)
![start --restore](https://user-images.githubusercontent.com/83953535/196144127-aa350f77-b88a-47eb-853c-7de5bc2ee0e3.gif)


p.s.
However when you stopped the bot again with `stop` command it will stop all orders (with "suspended")
![image](https://user-images.githubusercontent.com/83953535/196143744-6eb25b3c-c368-4970-a6a8-c296d80beb69.png)

**Comment by MementoRC** [comment]
@nikspz Correct, they are not placed in the `hanging orders`, but they are not `hanging orders` they are legitimate old pair of orders that behave like you described above. Your example shows a legitimate buy/sell pair, which is restored as buy/sell, the bot creates a new set of buy/sell on top of the restored buy/sell - The documentation shows 2 hanging orders, suspended then restored and showing up as hanging, but they were hanging prior to suspend command, thus my comment that this does not look like a bug, buy/sell pairs cannot be restored as 'hanging' - I could be missing something though

**Comment by nikspz** [comment]
Actual behavior:
bot NOT able to capture the orders left on exchnge at the event of `start --restore`
![start --restore for binance same result](https://user-images.githubusercontent.com/83953535/196635239-7691b699-fe7b-4f68-a483-ec3a89c5beb0.gif)



Expected Behavior:
1. Active orders left after the `exit --suspend` 
2. It labeled as "hanging" when start --restore is executed.
3. bot should be able to capture the orders left on exchnge at the event of `start --restore`

**Comment by MementoRC** [comment]
@nikspz That's understood, but the label "hanging" is related to PMM and Avellaneda only when the strategy selects `hanging_orders_enabled` is it not? I don't want to be prickly here, but make sure we have the same understanding, it seems like this is a feature request:
   - Capture the restored orders as ... "restored"
 I worked on the PMM hanging orders and the definition is a PMM orders pair that remains active when one-side only is executed, the other side is not removed, but kept hanging

**Comment by MementoRC** [comment]
@nikspz To be more specific, the code in PMM only set orders hanging IF hanging_order_enabled is True, but the orders are tracked:
```        if self._hanging_orders_enabled:
            # start tracking any restored limit order
            restored_order_ids = self.c_track_restored_orders(self.market_info)
            # make restored order hanging orders
            for order_id in restored_order_ids:
                order = next(o for o in self.market_info.market.limit_orders if o.client_order_id == order_id)
                if order:
                    self._hanging_orders_tracker.add_as_hanging_order(order)
```
So, do we want to remove the restriction? and verify that everything still works? Or do we want to have a new category `restored` for the labeling and display of the restored orders?

**Comment by cardosofede** [comment]
@rapcmia @MementoRC @nikspz from my side, I think that this feature is useless.
I don't know who wants to left the orders open in the market without doing anything.
To clarify this, I will suggest defining the expected behavior of the bot while using this feature (I think that @rapcmia has this clear) and make the code works like that.
Another solution can be remove this feature. If you want we can discuss this in our weekly dev call on Wednesday and vote there.

**Comment by rapcmia** [comment]
The Hummingbot Foundation team have removed the bounty on this ticket and decided to remove this feature

**Comment by rapcmia** [comment]
As part of the recent [priority poll](https://snapshot.org/#/hbot.eth/proposal/0x885b0ad9b47c645152bfd8a4603b3a8fd3ee07c0082ee5e8770f8368056f8286), we are opening the bounty again for this ticket:

Severity: P3
Bounty: 10,000 (https://hummingbot.org/maintenance/bugs/)
Rationale: To remove the `pause and resume` feature in Hummingbot Client

**Comment by klpanagi** [comment]
I would like this bounty to be assigned to me.
Already working on this...

**Comment by nikspz** [comment]
Assigning this issue to @klpanagi . Thank you for your contribution!

**Comment by klpanagi** [comment]
PR here: https://github.com/hummingbot/hummingbot/pull/6008

**Comment by nikspz** [comment]
Closing this ticket as paid. The PR has been deployed to development branch and will be available on the next release version 1.12.0.
